File: CSharp/Exercicio17.cs

public void SomaNaturais(int numero){
    int j = 0;
    for(int i=1; i <= numero; i++)
    {
        Console.Write($"{i} + ");
        j += i;
    }
    Console.Write($"= {j}");
}

SomaNaturais(5);

File: JavaScript/Exercicio17.js

function SomaNaturais(numero) {
    let j = 0;
    for(i=1; i<= numero; i++) {
        if(i < numero)
        process.stdout.write(`${i} + `)
        j += i;
    }
    process.stdout.write(`= ${j}`)
}

SomaNaturais(5);